core-web/libs/dot-primeng-theme-styles/src/scss/_variables.scss

```diff
@@ -20,6 +20,7 @@ $gray-hover: #eee;
 $deleted-content-color: #fdb8c0;
 $new-content-color: #ddffdd;
 $tr-hover-color: #e0e9f6;
+$blue-link: #1d4ed8;
```

Comment:
Probably the final design, this will be primary color.

Reply:
Yes, I know; we can leave it like this or rename it to something like `$experiments-links` ?

Reply:
Changed to $brand-primary.